fix(cli): do not remove @shopware-pwa packages

patzick opened the pull request:
## Changes

<!-- Describe the changes which you did and which issue you're closing
 example: closes #230
-->

closes #1789

we shouldn't remove @shopware-pwa packages if there're already some of them installed manually. Just update the version of nuxt-module which should be the same as CLI version.

originally proposed solution: #1790 - shoutout to @stigvanbrabant for pointing the problem

stigvanbrabant commented:
@patzick Yeah looks great to me! I don't know what the reasoning was for clearing the packages so i don't really know if it has other side effects tho.

patzick commented:
it was mostly for backward compatibility because at first we added all of the packages to `package.json` and at some point, we moved them as dependencies of `nuxt-module` so the upgrade is now iterating only one package in the simplest cases.

Nevertheless, I believe it's not needed anymore, especially if that causes problems like yours :)

mkucmus reviewed (approved):
you've removed the code that deleting. there is always a bigger fish 🐟
